index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <title>home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <img id="menu-id" src="./images/menu.png" alt="menu">
+        <div class="top-nav">
+            <ul>
+                <li><a href=""><img src="./images/facebook.png" alt=""></a></li>
+                <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+                <li><a href=""></a>English</li>
+                <li><a href=""></a>My page</li>
+                <li><a href=""></a>logout</li>
+            </ul>
+        </div>
+        <div class="down-nav">
+            <a href="index.html"><img src="./images/finalogo.png" alt=""></a>
+            <ul>
+                <li class="about-link"><a href="./about.html" >About</a></li>
+                <li><a href=""></a>Program</li>
+                <li><a href=""></a>Join</li>
+                <li><a href=""></a>Sponsor</li>
+                <li><a href=""></a>News</li>
+                <li class="campaign-link"><a href=""></a> contact</li>
+    
+            </ul>
+        </div>
+    </header>
+
+    <section class="wrapper">
+      
+        <div class="mobile-bar" >
+            <ul class="mobile-ul" >
+                <img id="close-btn" src="./images/close.png" alt="">
+                <li class="mobile-li"><a class="mobile-about" href="./about.html">About</a></li>
+                <li class="mobile-li"><a href=""></a>Program</li>
+                <li class="mobile-li"><a href=""></a>Join</li>
+                <li class="mobile-li"><a href=""></a>Sponsor</li>
+                <li class="mobile-li"><a href=""></a>News</li>
+                <li class="mobile-li"><a href=""></a> contact</li>
+            
+            </ul>
+        </div>
+        <div class="main-content">
+            <q class="quote">hello girls in TECH</q>
+            <h1 class="header-title">GIRLS IN TECH MEETUP 2023</h1>
```

Comment:
- [ ] Kindly make sure to increase your font size and use the right color for this title as it is on Cindy's design :recycle: 

- Your design :
![Screenshot from 2023-02-18 01-04-01](https://user-images.githubusercontent.com/97834160/219813771-5c509624-e784-434a-80b6-05c3c934b042.png)

- Behance:
![Screenshot from 2023-02-18 01-06-49](https://user-images.githubusercontent.com/97834160/219814101-7603e712-0ede-4482-88c2-e7860c89d175.png)